workshops/static/import_from_url.js

```diff
@@ -8,6 +8,7 @@ function import_from_url(url) {
     $("#id_start").val(data.start);
     $("#id_end").val(data.end);
     $("#id_reg_key").val(data.reg_key);
+    $("#id_tags").val(data.tags);
```

Comment:
On the import of a PyData conference, the `PyData` tag is selected.
@pbanaszkiewicz You may want to replicate it for SWC and DC event imports.